ncm-spma/src/main/pan/components/spma/yum/schema.pan

```diff
@@ -37,6 +37,8 @@ type component_spma_yum = {
     "run" ? legacy_binary_affirmation_string # Run the SPMA after configuring it
     "userpkgs_retry" : boolean = true
     "userpkgs" ? legacy_binary_affirmation_string # Allow user packages
+    @{ List of additional repo dirs to be included }
```

Comment:
make it more clear that this are additional repository directories that are not managed by ncm-spma. maybe replacing `additional` with `external`?